packages/dd-trace/src/exporters/agent/index.js

```diff
@@ -42,6 +42,12 @@ class AgentExporter {
   }
 
   export (spans) {
+    const { llmobs } = this._config
```

Comment:
Explaining this: we want `llmobs.agentlessEnabled` to have two uses (in parity with our existing Python SDK).
1. Use our agentless writer if specified (this is done in `llmobs/span_processor.js`)
2. Not export spans to the agent.

We expect (as has been so far with our Python SDK) a lot of new DD customers that aren't using APM. `agentlessEnabled` should allow customers to skip exporting spans to the agent. My question is, is it better to do it like this, or to create some kind of `LLMObsNoopExporter`, and have that be returned from `packages/dd-trace/src/exporter.js`?

Reply:
If the config was on the whole library, I'd go with some sort of `NoopExporter` or an `AgentlessExporter` that would be a noop while this is not supported for tracing. But the option being on `llmobs` specifically seems to me that we should just not do anything for tracing. If an agent is present, then traces will be reported as normal. If they aren't present, the HTTP connection will fail to open and nothing will be flushed (which is already roughly equivalent to noop). I don't think an option that is explicitly for one product should touch anything in other products as that is an unexpected side-effect.